telegram/_chat.py

```diff
@@ -2380,6 +2519,92 @@ async def copy_message(
             message_thread_id=message_thread_id,
         )
 
+    async def send_copies(
+        self,
+        from_chat_id: Union[str, int],
+        message_ids: Sequence[int],
+        disable_notification: ODVInput[bool] = DEFAULT_NONE,
+        protect_content: ODVInput[bool] = DEFAULT_NONE,
+        message_thread_id: Optional[int] = None,
+        remove_caption: ODVInput[bool] = DEFAULT_NONE,
+        *,
+        read_timeout: ODVInput[float] = DEFAULT_NONE,
+        write_timeout: ODVInput[float] = DEFAULT_NONE,
+        connect_timeout: ODVInput[float] = DEFAULT_NONE,
+        pool_timeout: ODVInput[float] = DEFAULT_NONE,
+        api_kwargs: Optional[JSONDict] = None,
+    ) -> Tuple["MessageId", ...]:
+        """Shortcut for::
+
+             await bot.copy_messages(chat_id=update.effective_chat.id, *args, **kwargs)
+
+        For the documentation of the arguments, please see :meth:`telegram.Bot.copy_messages`.
+
+        .. seealso:: :meth:`copy_message`, :meth:`send_copy`, :meth:`copy_messages`.
+
+        Returns:
+            Tuple[:class:`telegram.MessageId`]: On success, returns a tuple of `MessageId` of the
```

Comment:
```suggestion
            Tuple[:class:`telegram.MessageId`]: On success, returns a tuple of :class:`MessageId` of the
```

please also check in other places